### ISSUE TITLE
[Icon Request] Support for Sparql .rq files

### ISSUE THREAD

**Issue by Jefwillems** [vscode-icons/vscode-icons#1814]
- [x] I'm sure this issue is not a _duplicate_.
- [x] I'm submitting an icon request:
  - Type: `extension`
  - Icon Name: `sparql`
  - Sample original Icon: N/A
  - Extensions: `.rq`

- More info: [.rq](https://www.filesuffix.com/en/extension/rq) file extension

JimiC: Possible `vscode` syntax highlighting [extension](https://marketplace.visualstudio.com/items?itemName=stardog-union.stardog-rdf-grammars)

**Comment by KingDarBoja** [comment]
After a quick search, I found this: [W3C Semantic Web Logos and Policies](https://www.w3.org/2007/10/sw-logos.html).

So no idea how the logo design could be since the cube must be provided along with the "SPARQL" TextBox.

**Comment by JimiC** [comment]
Yeah, difficulties here. At some point I want to support also `RDF`, as we use it at work, along with `GLM` (Gridlab-d). Currently I have no clue on how to tackle this.

**Comment by KingDarBoja** [comment]
As stated on the docs:

_"Not-for-profit use: For the Technology Buttons, logo size, foreground, and background colors of the technology name MAY be modified without permission."_

So I had in mind using the text just below the cube so we don't have to make big changes to the original logo. We can use one of the background colors provided for the TexBox, like the purple or gray one to avoid dealing with light/dark versions. What do you think @JimiC ?

**Comment by JimiC** [comment]
Nope. Not fond of using text as an icon.

**Comment by KingDarBoja** [comment]
@JimiC What if we just change the cube colors? On the FAQ section, item 3: 

> **3. May I change the cube colors**
> _If there is a "W3C" in the logo, no. Otherwise, yes._

**Comment by JimiC** [comment]
Still doesn't help. It will come to me eventually.
